fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -260,12 +260,6 @@ abstract class BaseBrowserFragment :
         val originalContext = ActivityContextWrapper.getOriginalContext(activity)
         binding.engineView.setActivityContext(originalContext)
 
-        browserFragmentStore = StoreProvider.get(this) {
-            BrowserFragmentStore(
-                BrowserFragmentState(),
-            )
-        }
-
```

Comment:
There are some awkward lifecycle constraints here where the `BrowserToolbarView` is recreated as the fragment is, whereas this store would survive. Since I copied the existing CFR handling directly, I did not think through the correct way to handle lifecycle management if this Store actually survived configuration changes.